client/layout/404.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+
+let Page404 = React.createClass( {
+	displayName: 'Page404',
+
+	statics: {
+		show() {
+			ReactDOM.render(
+				React.createElement( Page404, {} ),
+				document.getElementById( 'primary' )
+			);
+		}
+	},
+
+	redirectHome() {
+		page.redirect( '/' );
+	},
+
+	render() {
+		return (
+			<EmptyContent
+				illustration="/calypso/images/drake/drake-404.svg"
+				title={ this.translate( 'Uh oh. Page not found.' ) }
+				line={ this.translate( 'Sorry, the page you were looking for doesn\'t exist or has been moved.' ) }
+				action={ this.translate( 'Go back home' ) }
```

Comment:
`Go back home` seems a bit... rude :laughing: With lower case `home` it seems like you are saying "Go back home, kid, you're not welcome here" :stuck_out_tongue_winking_eye: `Return to Home` or `Return to My Sites` or something more specific could be better.